data.py

```diff
@@ -13,11 +16,44 @@ def load() -> tuple[dict[str, dict], dict[str, str]]:
 	text: dict[str, str] = {}
 	with open('data/aura_skill.json', 'rb') as f:
 		raw_text: list[dict] = json.load(f)
-	prefixes = ['You and nearby', 'Aura grants', 'Buff grants']
+	prefixes = ['You and nearby', 'Your and nearby', 'Aura grants', 'Buff grants']
 	for translation in raw_text:
 		for k in translation['ids']:
 			translated = translation['English'][0]
 			if any(translated['string'].startswith(prefix + ' ') for prefix in prefixes):
 				text[k] = translated
 
 	return gems, text
+
+def legion_passive_mapping() -> dict:
+	""" Maps names of timeless legion passives to their effects """
+	# I couldn't find any of this info in the RePoE data, so I'm grabbing it from the path of building repo
+	with open('data/LegionPassives.lua', 'r') as file:
+		content = file.read()
+		content = re.sub(r'\[(\d+)\] =', r'["\1"] =', content)  # turns integer keys into strings
```

Comment:
this whole thing is mad genius. I hate it, but it's great
this in particular perhaps could be replaced by just `return {int(node['dn']): ...`

Reply:
trust me i hate it too lol.
node['dn'] is actually just the name of the alternative notable, not an integer. 
we need the names for the mapping here since the timeless jewel csv files only give the names of the legion notables